xds/src/test/java/io/grpc/xds/LocalityStoreTest.java

```diff
@@ -274,8 +278,17 @@ public PickResult pickSubchannel(PickSubchannelArgs args) {
     verify(random, never()).nextInt(1000_000);
   }
 
+  @Test
+  public void updateLoaclityStore_withEmptyDropList() {
+    localityStore.updateDropPercentage(ImmutableList.<DropOverload>of());
```

Comment:
Isn't empty list the default value already? Should `updateLoaclityStore()` be renamed to `updateLoaclityStore_withEmptyDropList()`?